pacemaker/pacemaker.go

```diff
@@ -38,8 +38,9 @@ func NewFixedLeader(hs *hotstuff.HotStuff, leaderID hotstuff.ReplicaID) *FixedLe
 // Run runs the pacemaker which will beat when the previous QC is completed
 func (p FixedLeader) Run(ctx context.Context) {
 	notify := p.GetNotifier()
+	dummyCtx := context.Background()
```

Comment:
I think that using a local variable to store the background context is unnecessary. Just call `context.Background()` in both calls to `Propose`